book/validation.rst

```diff
@@ -391,10 +395,13 @@ constraint, have several configuration options available. Suppose that the
 
             public static function loadValidatorMetadata(ClassMetadata $metadata)
             {
-                $metadata->addPropertyConstraint('gender', new Choice(array(
```

Comment:
Yes, at the moment there are not horizontal scrollbars, but after PR 4779 will be applied the `Choice` will become `Assert\Choice` which will make the line too long and it's a 99% chance it will lead to horizontal scrollbar.